Add documentation for renault.ac_set_schedules

vhkristof opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
This PR adds documentation for the new service that is implemented by https://github.com/home-assistant/core/pull/125006

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125006
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit


- **New Features**
	- Introduced a new action to update air conditioning schedules for vehicles, allowing users to set multiple schedules with enhanced flexibility.
	- Documentation now includes examples to aid user understanding of the new scheduling functionality.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

c0ffeeca7 committed:
Rename yaml icon to config file icon. update description (#34193)

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

override80 committed:
Fix: update seventeentrack example card (#34198)

c0ffeeca7 committed:
Matter device update: add screenshot (#34197)

starkillerOG committed:
Explain Reolink password character set limitiations (#34196)

Co-authored-by: Franck Nijhof <git@frenck.dev>

frenck committed:
Fix/Adjust WLED documentation (#34204)

Commit:
Improve documentation for IronOS integration (#34094)

* Clarification for raw tip voltage sensor, add ha category

* add comma

googanhiem committed:
Upload of the Works with Home Assistant blog (#34206)

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.11 to 0.17.12 (#34217)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dudau16 committed:
Fix documentation YAML example in androidtv_remote (#34228)

dependabot[bot] committed:
build(deps-dev): bump parser from 3.3.4.1 to 3.3.4.2 (#34218)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump parallel from 1.25.1 to 1.26.1 (#34216)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

starkillerOG committed:
Add Reolink doorbell white to supported list (#34214)

frenck committed:
Fix/adjust service -> action in notification groups (#34229)

serl committed:
Flux: Specify default values for start_time and stop_time (#34235)

frenck committed:
2024.8.1 (#34236)

Co-authored-by: TheJulianJES <TheJulianJES@users.noreply.github.com>

bieniu committed:
Add information about minimal firmware version for Shelly virtual components (#34243)

DubhAd committed:
Update sensor.mqtt.markdown (#34209)

Closing quote missing from the template

deviantintegral committed:
Clarify continuous mode is enabled by default (#34237)

tronikos committed:
Update androidtv_remote.markdown (#34239)

jbouwh committed:
Update IMAP docs on use of App passwords and removed Microsoft support (#34244)

* Update IMAP docs on use of App passwords and removed Microsoft support

* Suggested update

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Tiny tweak

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11517 to 0.5.11518 (#34251)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rexml from 3.3.4 to 3.3.5 (#34253)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump parallel from 1.26.1 to 1.26.2 (#34250)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump mustermann from 3.0.1 to 3.0.2 (#34249)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump concurrent-ruby from 1.3.3 to 1.3.4 (#34252)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Hedda committed:
Link to all entities that is support in group (#34208)

* Link to all entities that is support in group

Link to all entities that are currently supported in a group

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Rename service-calls file to perform-actions to align terminology (#34254)

jbouwh committed:
Cleanup MQTT legacy notify instruction using mqtt publish (#34260)

jbouwh committed:
Cleanup mqtt publish action template option documentation (#34257)

* Cleanup mqtt publish action template option documentation

* Missed instance

miaucl committed:
Remove deprecated information from Swiss public transport (#34262)

c0ffeeca7 committed:
Assist: update ATOM Echo tutorial (#34263)

c0ffeeca7 committed:
Core installation: add step to check for correct python version (#34264)

c0ffeeca7 committed:
S3 box update screenshots and steps (#34259)

c0ffeeca7 committed:
Raspberri Pi installation: re-add links to HA image (#34265)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11518 to 0.5.11519 (#34276)

Dawidooox committed:
Correct new Wake on LAN device type section (button) (#34274)

c0ffeeca7 committed:
Add termref to configuration.yaml (#34275)

- to make it clearer that the example yaml snipped belongs there

c0ffeeca7 committed:
Badges: add screencast on adding a badge (#34280)

remy committed:
use html5 details element instead of javascript (#34279)

noahhusby committed:
Add note regarding device support for Russound integrations (#34183)

* refact: add note regarding device support for Russound integrations

* refact: replace device support notes with sections

* Tiny markdown tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
integration term definition: change link to point to more detailed description (#34284)

zubair-io committed:
Fixing a link on zha.markdown (#34299)

The Link to the Zigbee binding section needed to be updated

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.12 to 0.17.13 (#34307)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11519 to 0.5.11520 (#34306)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Glossary: add term 'custom integration' (#34283)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11520 to 0.5.11525 (#34320)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

frenck committed:
2024.8.2 (#34328)

ludeeus committed:
Fix generic-x86-64 OS download URL (#34351)

farmio committed:
Fix KNX telegram trigger data (#34333)

* Fix KNX telegram trigger data

and add example use in template

* escape template

Anrijs committed:
added aranet radiation sensor to supported devices (#34360)

NYZack committed:
Update badges.markdown (#34314)

The description of the "conditions" behavior seems to be false.  "conditions" seems to exhibit "and", not "or" behavior, with respect to each condition.

Bre77 committed:
Update tesla_fleet.markdown (#34304)

Commit:
Add time to today_at() functions (#34310)

Commit:
default for today_at() (#34311)

merlinacious committed:
Updated hue.markdown for v1 hue bridges (#34288)

* Updated hue.markdown for v1 hue bridges 

Added specific instructions on how to set Scenes for v1 hue bridges.   Had to do a lot of research to find these as a lot of the current documentation and videos are for v2 hue bridges.

* Tiny style tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump parallel from 1.26.2 to 1.26.3 (#34362)

Bumps [parallel](https://github.com/grosser/parallel) from 1.26.2 to 1.26.3.
- [Commits](https://github.com/grosser/parallel/compare/v1.26.2...v1.26.3)

---
updated-dependencies:
- dependency-name: parallel
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop-ast from 1.32.0 to 1.32.1 (#34363)

Bumps [rubocop-ast](https://github.com/rubocop/rubocop-ast) from 1.32.0 to 1.32.1.
- [Release notes](https://github.com/rubocop/rubocop-ast/releases)
- [Changelog](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-ast/compare/v1.32.0...v1.32.1)

---
updated-dependencies:
- dependency-name: rubocop-ast
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.13 to 0.17.14 (#34375)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.17.13 to 0.17.14.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.17.13...v0.17.14)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11525 to 0.5.11531 (#34376)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11525 to 0.5.11531.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump textlint from 14.0.5 to 14.2.0 (#34365)

Bumps [textlint](https://github.com/textlint/textlint) from 14.0.5 to 14.2.0.
- [Release notes](https://github.com/textlint/textlint/releases)
- [Commits](https://github.com/textlint/textlint/compare/v14.0.5...v14.2.0)

---
updated-dependencies:
- dependency-name: textlint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Raspberry Pi: rephrase A2 class requirement (#34379)

* Raspberry Pi: rephrase A2 class requirement

* Update source/installation/raspberrypi.markdown

Co-authored-by: Stefan Agner <stefan@agner.ch>

---------

Co-authored-by: Stefan Agner <stefan@agner.ch>

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.14 to 0.17.15 (#34388)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.17.14 to 0.17.15.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.17.14...v0.17.15)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11531 to 0.5.11532 (#34389)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11531 to 0.5.11532.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Backup: add link to backup procedure for Core and Container (#34391)

* Backup: add link to backup procedure for Core and Container

* Fix link

googanhiem committed:
Upload blog for HACS 2.0 (#34390)

* Upload blog for HACS 2.0

* Numerous fixes

jfurtner committed:
Update blueprint schema documentation (#34395)

* Update blueprint schema documentation

min_version documentation doesn't mention required format, make that explicit in doc and in note about sections

* Update source/_docs/blueprint/schema.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Update source/_docs/blueprint/schema.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

googanhiem committed:
Fix opengraph in HACS blog (#34393)

civita committed:
Update tailscale.markdown (#34346)

dependabot[bot] committed:
build(deps-dev): bump rexml from 3.3.5 to 3.3.6 (#34397)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11532 to 0.5.11535 (#34396)

ludeeus committed:
Update URLs pointing to wrong domain (#34398)

c0ffeeca7 committed:
Fix typo (#34400)

dependabot[bot] committed:
build(deps-dev): bump rbs from 3.5.2 to 3.5.3 (#34412)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11535 to 0.5.11537 (#34411)

karwosts committed:
Update badges.markdown (#34431)

kamiyo committed:
Update nws.markdown (#34428)

Fix wrong hyperlinks to weather.get_forecasts, as pointed out by #34424

balloob committed:
2024.8.3 (#34439)

* 2024.8.3

* Uno mas

* Add some whitespace back

frenck committed:
Fix 2024.8.3 patch release notes (#34440)

mekaneck committed:
Clarify when numeric state will update when other entities are used as limits (#34335)

ThoStaKa committed:
Update tellduslive.markdown (#34066)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11537 to 0.5.11545 (#34447)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11537 to 0.5.11545.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

jeeftor committed:
adding to financial status (#34330)

balloob committed:
Update Cast splash image (#34448)

IceBotYT committed:
Clarify Linear Garage Door integration, notify of new integration coming soon (#34402)

* Clarify Linear Garage Door integration, notify of new integration coming soon

* Small tweak

* Tweaks

* Remove redundant 'for more info'

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.15 to 0.17.16 (#34455)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.17.15 to 0.17.16.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.17.15...v0.17.16)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11545 to 0.5.11547 (#34456)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11545 to 0.5.11547.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

emontnemery committed:
Clarify inputs in blueprint sections are flattened (#34457)

* Clarify inputs in blueprint sections are flattened

* Apply suggestions from code review

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

unfug-at-github committed:
Update statistics.markdown to give more details about time based average computation (#34451)

* Update statistics.markdown

Added some more details regarding the time based averages computations

* Update statistics.markdown fixed lint issues

* Update statistics.markdown fixed table format

* Update statistics.markdown further AI recommendations

dependabot[bot] committed:
build(deps-dev): bump google-protobuf from 4.27.3 to 4.27.4 (#34468)

Bumps [google-protobuf](https://github.com/protocolbuffers/protobuf) from 4.27.3 to 4.27.4.
- [Release notes](https://github.com/protocolbuffers/protobuf/releases)
- [Changelog](https://github.com/protocolbuffers/protobuf/blob/main/protobuf_release.bzl)
- [Commits](https://github.com/protocolbuffers/protobuf/commits)

---
updated-dependencies:
- dependency-name: google-protobuf
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11547 to 0.5.11549 (#34469)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11547 to 0.5.11549.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ehcloninger committed:
Notes on integrating SleepIQ with pihole on the network. (#33061)

* Notes on integrating SleepIQ with pihole on the network.

* Apply suggestions from code review

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Move config flow and prereqs (#34471)

* Move config flow and prereqs up

* Fix terminology

HarvsG committed:
add 'hours' to the prompt (#34474)

b-uwe committed:
Add HAshop to the list of Green distributors (#34472)

allenporter committed:
Add instructions for using ollama experimental home control (#34465)

* Add instructions for using ollama experimental home control

* Update source/_integrations/ollama.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Update source/_integrations/ollama.markdown

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

bramkragten committed:
Merge branch 'current' into rc

bramkragten committed:
Merge branch 'next' into rc

frenck committed:
2024.9: Beta release notes (#34194)

* 2024.9: Beta release notes

* Adjust milestones

* Removal of asterisk integrations

* Add proximity feature deprecation

* Add Anthropic integration

* Add initial notes

* Update 2024-09-04-release-20249.markdown

* temp screenshots

* Update 2024-09-04-release-20249.markdown

---------

Co-authored-by: Bram Kragten <mail@bramkragten.nl>

bramkragten committed:
Fix release note images (#34483)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11549 to 0.5.11551 (#34487)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11549 to 0.5.11551.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump google-protobuf from 4.27.4 to 4.28.0 (#34488)

Bumps [google-protobuf](https://github.com/protocolbuffers/protobuf) from 4.27.4 to 4.28.0.
- [Release notes](https://github.com/protocolbuffers/protobuf/releases)
- [Changelog](https://github.com/protocolbuffers/protobuf/blob/main/protobuf_release.bzl)
- [Commits](https://github.com/protocolbuffers/protobuf/commits)

---
updated-dependencies:
- dependency-name: google-protobuf
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

iomartin committed:
Add limitations section to Enmax integration (#34493)

Enmax Energy only publishes monthly data via Opower, make a note in the documentation

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.17.16 to 0.17.17 (#34496)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.17.16 to 0.17.17.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.17.16...v0.17.17)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11551 to 0.5.11553 (#34497)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11551 to 0.5.11553.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Locked out: add screencast of auth reset (#34499)

* Locked out: add screencast of auth reset

* Update steps to point to Green and Yellow reset procedures

* Fix copy paste error

* Update clip

* Fix video to show standard port

balloob committed:
Add stream to 2024.9 post

vhkristof committed:
Add documentation for renault.ac_set_schedules

Sergeus committed:
Fix typo in "default sentences" documentation (#34528)

matthiasbeyer committed:
Add missing quote (#34519)

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

mvn23 committed:
Fix opentherm_gw gpio modes (#34501)

mxr committed:
home connect: Update links for registering an app and logging out (#34508)

* Update links for registering an app and logging out

* Update link

* Update source/_integrations/home_connect.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* tiny tweaks

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop from 1.65.1 to 1.66.0 (#34535)

Bumps [rubocop](https://github.com/rubocop/rubocop) from 1.65.1 to 1.66.0.
- [Release notes](https://github.com/rubocop/rubocop/releases)
- [Changelog](https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop/compare/v1.65.1...v1.66.0)

---
updated-dependencies:
- dependency-name: rubocop
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump logger from 1.6.0 to 1.6.1 (#34536)

Bumps [logger](https://github.com/ruby/logger) from 1.6.0 to 1.6.1.
- [Release notes](https://github.com/ruby/logger/releases)
- [Commits](https://github.com/ruby/logger/compare/v1.6.0...v1.6.1)

---
updated-dependencies:
- dependency-name: logger
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

mxr committed:
home connect: Add instructions for getting the client ID and secret (#34509)

* home connect: Add instructions for getting the client ID and secret

* Update home_connect.markdown

* Update home_connect.markdown

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

kylehakala committed:
Document prerequisites for Webmin integration (#34515)

* Update webmin.markdown

Add details about prerequisites (installing XML::Parser), add additional context for user account creation, update phrasing of existing content

* Minor tweaks for verbiage, proper markdown

* Additional readability improvements for Webmin

* Fix odd wording in Webmin account benefits

"allows for" might imply that a separate account is automatically more secure; switching to "can allow for" to instead imply that things can be made more granular if desired.

* Update source/_integrations/webmin.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Tiny style tweaks

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

googanhiem committed:
Small additions to beta release notes (#34510)

renekliment committed:
ubus.markdown: fix JSON syntax (#34522)

darkrain-nl committed:
Update unifiprotect.markdown with information about event for doorbell (#34537)

* Update unifiprotect.markdown with information about event for doorbell

Add more information about the event entity when a doorbell is rang.

* Tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 commented on source/_integrations/renault.markdown:
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
+
+Notes:
+
+- `schedules` can be in the form `{'id':1,...}` when updating a single schedules, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call
```
```suggestion
- `schedules` can be in the form `{'id':1,...}` when updating a single schedule, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call.
```

c0ffeeca7 commented on source/_integrations/renault.markdown:
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
+Notes:
+
+- `schedules` can be in the form `{'id':1,...}` when updating a single schedules, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call
+- the `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle)
```
```suggestion
- The `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle).
```

c0ffeeca7 commented on source/_integrations/renault.markdown:
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
+
+- `schedules` can be in the form `{'id':1,...}` when updating a single schedules, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call
+- the `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle)
+- the `activated` flag is an optional boolean. If it is not provided, then the existing flag will be kept as is.
```
```suggestion
- The `activated` flag is an optional boolean. If it is not provided, then the existing flag will be kept as is.
```

c0ffeeca7 commented on source/_integrations/renault.markdown:
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
+- `schedules` can be in the form `{'id':1,...}` when updating a single schedules, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call
+- the `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle)
+- the `activated` flag is an optional boolean. If it is not provided, then the existing flag will be kept as is.
+- the `monday` to `sunday` elements are optional. If they are not provided, then the existing settings will be kept for each day. If they are provided as None, then the existing setting will be cleared. If a value is provided, it must conform to this format `{'readyAtTime':'T12:00Z'}` where start time is in UTC format and the duration is in minutes.
```
```suggestion
- The `monday` to `sunday` elements are optional. If they are not provided, then the existing settings will be kept for each day. If they are provided as None, then the existing setting will be cleared. If a value is provided, it must conform to this format `{'readyAtTime':'T12:00Z'}` where start time is in UTC format and the duration is in minutes.
```

c0ffeeca7 committed:
Tiny style tweaks

c0ffeeca7 reviewed (dismissed):
Thank you, @vhkristof :+1:

catsmanac committed:
Add Enphase_envoy prerequisites (#34539)

* Add Enphase_envoy prerequisites

* Implement review suggestions

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Remove links to custom integrations

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop-ast from 1.32.1 to 1.32.2 (#34547)

Bumps [rubocop-ast](https://github.com/rubocop/rubocop-ast) from 1.32.1 to 1.32.2.
- [Release notes](https://github.com/rubocop/rubocop-ast/releases)
- [Changelog](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-ast/compare/v1.32.1...v1.32.2)

---
updated-dependencies:
- dependency-name: rubocop-ast
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Enable entity (#34550)

* Enable a single entity from device page

* Add related topic

* Add term ref

* Add missing step

* Add some background info why entities are disabled by default

* Rephrase

* Fix typo

luca-angemi committed:
Fix typo in google_translate (#34552)

joostlek committed:
New WWHA blog upload (#34551)

* New WWHA blog upload

* Update source/_posts/2024-09-03-aqara-joins-works-with-home-assistant.markdown

---------

Co-authored-by: Gord <31004434+googanhiem@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11553 to 0.5.11554 (#34561)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11553 to 0.5.11554.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump mustermann from 3.0.2 to 3.0.3 (#34562)

Bumps [mustermann](https://github.com/sinatra/mustermann) from 3.0.2 to 3.0.3.
- [Commits](https://github.com/sinatra/mustermann/compare/v3.0.2...v3.0.3)

---
updated-dependencies:
- dependency-name: mustermann
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rexml from 3.3.6 to 3.3.7 (#34563)

Bumps [rexml](https://github.com/ruby/rexml) from 3.3.6 to 3.3.7.
- [Release notes](https://github.com/ruby/rexml/releases)
- [Changelog](https://github.com/ruby/rexml/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/rexml/compare/v3.3.6...v3.3.7)

---
updated-dependencies:
- dependency-name: rexml
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.77.8 to 1.78.0 (#34564)

Bumps [sass-embedded](https://github.com/sass-contrib/sass-embedded-host-ruby) from 1.77.8 to 1.78.0.
- [Commits](https://github.com/sass-contrib/sass-embedded-host-ruby/compare/v1.77.8...v1.78.0)

---
updated-dependencies:
- dependency-name: sass-embedded
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Event docs and integration: add related links (#34569)

c0ffeeca7 committed:
State trigger: rephrase (#34571)

* State trigger: rephrase

* Add intro text to example

c0ffeeca7 committed:
Light: add section on states (#34573)

bramkragten committed:
Release notes 2024.9 (#34575)

* build(deps-dev): bump sorbet-runtime from 0.5.11549 to 0.5.11551 (#34487)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11549 to 0.5.11551.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps-dev): bump google-protobuf from 4.27.4 to 4.28.0 (#34488)

Bumps [google-protobuf](https://github.com/protocolbuffers/protobuf) from 4.27.4 to 4.28.0.
- [Release notes](https://github.com/protocolbuffers/protobuf/releases)
- [Changelog](https://github.com/protocolbuffers/protobuf/blob/main/protobuf_release.bzl)
- [Commits](https://github.com/protocolbuffers/protobuf/commits)

---
updated-dependencies:
- dependency-name: google-protobuf
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Add limitations section to Enmax integration (#34493)

Enmax Energy only publishes monthly data via Opower, make a note in the documentation

* build(deps-dev): bump ruby-lsp from 0.17.16 to 0.17.17 (#34496)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.17.16 to 0.17.17.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.17.16...v0.17.17)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps-dev): bump sorbet-runtime from 0.5.11551 to 0.5.11553 (#34497)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11551 to 0.5.11553.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Locked out: add screencast of auth reset (#34499)

* Locked out: add screencast of auth reset

* Update steps to point to Green and Yellow reset procedures

* Fix copy paste error

* Update clip

* Fix video to show standard port

* Fix typo in "default sentences" documentation (#34528)

* Add missing quote (#34519)

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

* Fix opentherm_gw gpio modes (#34501)

* home connect: Update links for registering an app and logging out (#34508)

* Update links for registering an app and logging out

* Update link

* Update source/_integrations/home_connect.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* tiny tweaks

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* build(deps-dev): bump rubocop from 1.65.1 to 1.66.0 (#34535)

Bumps [rubocop](https://github.com/rubocop/rubocop) from 1.65.1 to 1.66.0.
- [Release notes](https://github.com/rubocop/rubocop/releases)
- [Changelog](https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop/compare/v1.65.1...v1.66.0)

---
updated-dependencies:
- dependency-name: rubocop
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps-dev): bump logger from 1.6.0 to 1.6.1 (#34536)

Bumps [logger](https://github.com/ruby/logger) from 1.6.0 to 1.6.1.
- [Release notes](https://github.com/ruby/logger/releases)
- [Commits](https://github.com/ruby/logger/compare/v1.6.0...v1.6.1)

---
updated-dependencies:
- dependency-name: logger
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* home connect: Add instructions for getting the client ID and secret (#34509)

* home connect: Add instructions for getting the client ID and secret

* Update home_connect.markdown

* Update home_connect.markdown

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Document prerequisites for Webmin integration (#34515)

* Update webmin.markdown

Add details about prerequisites (installing XML::Parser), add additional context for user account creation, update phrasing of existing content

* Minor tweaks for verbiage, proper markdown

* Additional readability improvements for Webmin

* Fix odd wording in Webmin account benefits

"allows for" might imply that a separate account is automatically more secure; switching to "can allow for" to instead imply that things can be made more granular if desired.

* Update source/_integrations/webmin.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Tiny style tweaks

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* ubus.markdown: fix JSON syntax (#34522)

* Update unifiprotect.markdown with information about event for doorbell (#34537)

* Update unifiprotect.markdown with information about event for doorbell

Add more information about the event entity when a doorbell is rang.

* Tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Add Enphase_envoy prerequisites (#34539)

* Add Enphase_envoy prerequisites

* Implement review suggestions

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Remove links to custom integrations

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* build(deps-dev): bump rubocop-ast from 1.32.1 to 1.32.2 (#34547)

Bumps [rubocop-ast](https://github.com/rubocop/rubocop-ast) from 1.32.1 to 1.32.2.
- [Release notes](https://github.com/rubocop/rubocop-ast/releases)
- [Changelog](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-ast/compare/v1.32.1...v1.32.2)

---
updated-dependencies:
- dependency-name: rubocop-ast
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Enable entity (#34550)

* Enable a single entity from device page

* Add related topic

* Add term ref

* Add missing step

* Add some background info why entities are disabled by default

* Rephrase

* Fix typo

* Fix typo in google_translate (#34552)

* New WWHA blog upload (#34551)

* New WWHA blog upload

* Update source/_posts/2024-09-03-aqara-joins-works-with-home-assistant.markdown

---------

Co-authored-by: Gord <31004434+googanhiem@users.noreply.github.com>

* Split august and yale integrations (#33890)

* Add yale integration

This PR should not merge until the integration branch
in https://github.com/home-assistant/core/pull/122253 is complete

* Update source/_integrations/yale.markdown

* Tiny tweaks

* Apply suggestions from code review

* review comments

* Apply suggestions from code review

* Update source/_integrations/yale.markdown

* Update source/_integrations/yale.markdown

* Apply suggestions from code review

* Update source/_integrations/yale.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Simple Fin - Add Problem Sensor Documentation (#33979)

* Adding the problems ensor

* Tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Redirect virtual integration yale_home to point to yale (#34482)

* Remove Yale (Assa Abloy) devices from the august integration docs (#34484)

* Remove Yale (Assa Abloy) devices from the august integration docs

The new yale integration should be used for these devices instead

* Remove Assa Abloy note since August is now owned by FBIN

* Change entity badge display type to 3 booleans : name, state and icon (#34503)

* Remove KNX free-online-training paragraph (#34531)

* build(deps-dev): bump sorbet-runtime from 0.5.11553 to 0.5.11554 (#34561)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11553 to 0.5.11554.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps): bump mustermann from 3.0.2 to 3.0.3 (#34562)

Bumps [mustermann](https://github.com/sinatra/mustermann) from 3.0.2 to 3.0.3.
- [Commits](https://github.com/sinatra/mustermann/compare/v3.0.2...v3.0.3)

---
updated-dependencies:
- dependency-name: mustermann
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps-dev): bump rexml from 3.3.6 to 3.3.7 (#34563)

Bumps [rexml](https://github.com/ruby/rexml) from 3.3.6 to 3.3.7.
- [Release notes](https://github.com/ruby/rexml/releases)
- [Changelog](https://github.com/ruby/rexml/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/rexml/compare/v3.3.6...v3.3.7)

---
updated-dependencies:
- dependency-name: rexml
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps-dev): bump sass-embedded from 1.77.8 to 1.78.0 (#34564)

Bumps [sass-embedded](https://github.com/sass-contrib/sass-embedded-host-ruby) from 1.77.8 to 1.78.0.
- [Commits](https://github.com/sass-contrib/sass-embedded-host-ruby/compare/v1.77.8...v1.78.0)

---
updated-dependencies:
- dependency-name: sass-embedded
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* Event docs and integration: add related links (#34569)

* State trigger: rephrase (#34571)

* State trigger: rephrase

* Add intro text to example

* Light: add section on states (#34573)

* Update release notes for 2024.9

* Update 2024-09-04-release-20249.markdown

* Update 2024-09-04-release-20249.markdown

* Update 2024-09-04-release-20249.markdown

* Update 2024-09-04-release-20249.markdown

* update images

* address comments

* Update 2024-09-04-release-20249.markdown

* Apply suggestions from code review

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* suggestions

* Apply suggestions from code review

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

---------

Signed-off-by: dependabot[bot] <support@github.com>
Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: iomartin <iomartin@users.noreply.github.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>
Co-authored-by: Stuart Holland <soulc3@gmail.com>
Co-authored-by: Matthias Beyer <mail@beyermatthias.de>
Co-authored-by: mvn23 <schopdiedwaas@gmail.com>
Co-authored-by: Max R <mxr@users.noreply.github.com>
Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: kylehakala <kylehakala@users.noreply.github.com>
Co-authored-by: René Kliment <rene@renekliment.cz>
Co-authored-by: darkrain-nl <24763370+darkrain-nl@users.noreply.github.com>
Co-authored-by: Arie Catsman <120491684+catsmanac@users.noreply.github.com>
Co-authored-by: Luca Angemi <luca.angemi@gmail.com>
Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>
Co-authored-by: Gord <31004434+googanhiem@users.noreply.github.com>
Co-authored-by: J. Nick Koston <nick@koston.org>
Co-authored-by: Jeef <jeeftor@users.noreply.github.com>
Co-authored-by: Paul Bottein <paul.bottein@gmail.com>
Co-authored-by: Matthias Alphart <farmio@alphart.net>

bramkragten committed:
Merge branch 'current' into release

bramkragten committed:
merge current in rc (#34577)

bramkragten committed:
2024.9 (#34578)

farmio committed:
Remove KNX last release item from current release notes (#34583)

balloob committed:
Fix TOC

sdb9696 committed:
Update tplink docs regarding Unavailable entities (#34505)

Co-authored-by: J. Nick Koston <nick@koston.org>

tl-sl committed:
Add note about required core firmware version (#34591)

dependabot[bot] committed:
build(deps-dev): bump rubocop from 1.66.0 to 1.66.1 (#34594)

Bumps [rubocop](https://github.com/rubocop/rubocop) from 1.66.0 to 1.66.1.
- [Release notes](https://github.com/rubocop/rubocop/releases)
- [Changelog](https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop/compare/v1.66.0...v1.66.1)

---
updated-dependencies:
- dependency-name: rubocop
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
Update note on Thread on Connect ZBT-1 page (#34593)

c0ffeeca7 committed:
Explain that event and event entity are not the same (#34574)

* Explain that event and event entity are not the same

* Implement review feedback

* Fix typo

* Update source/_docs/configuration/events.markdown

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

---------

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11554 to 0.5.11557 (#34610)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11554 to 0.5.11557.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

balloob committed:
2024.9.1 (#34617)

* 2024.9.1

* Update source/_posts/2024-09-04-release-20249.markdown

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

---------

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

vhkristof committed:
Merge branch 'home-assistant:current' into add-documentation-for-renault.ac_set_schedules

vhkristof committed:
Merge branch 'next' into add-documentation-for-renault.ac_set_schedules

vhkristof commented:
Will create a new PR. Messed up with wrong merge :-/

vhkristof commented:
See #34636
